src/Tesseract/ResultIterator.cs

```diff
@@ -34,41 +34,67 @@ public string GetText(PageIteratorLevel level)
                 return null;
             }
 
-            bool is_bold, is_italic, is_underlined,
```

Comment:
Assuming that the cache is moved to Result Iterator maybe something like (in pesudo code):

```
IntPtr nameHandle = Interop.TessApi.ResultIteratorWordFontAttributes(handle, ...attributes, out fontId);
FontInfo fontInfo;
if(!_cache.TryGetValue(fontId, out fontInfo)) {
     string fontName = MarshalHelper.PtrToString(nameHandle, TextEncoding.UTF8);
     fontInfo = new FontInfo(fontName, isItalic, isBold, isMonospace, isSerif);
     _cache.Add(fontId, fontInfo);
}
return new FontAttributes(fontInfo, isUnderlined, isSmallcaps, pointsize);